chart/customer/templates/server.yaml

```diff
@@ -79,8 +79,12 @@ spec:
             value: "https://{{ .Release.Name }}-{{ .Values.service.auth }}:9443/oidc/endpoint/OP/jwk"
           - name: jwksIssuer
             value: "https://{{ .Release.Name }}-{{ .Values.service.auth }}:9443/oidc/endpoint/OP"
+          - name: zipkinHost
+            value: "{{ .Release.Name }}-{{ .Values.service.zipkinHost }}"
+          - name: zipkinPort
+            value: "{{ .Values.service.zipkinPort }}"
           - name: administratorRealm
-            value: user:https://{{ .Release.Name }}-{{ .Values.service.auth }}:9443/oidc/endpoint/OP/user
+            value: "user:https://{{ .Release.Name }}-{{ .Values.service.auth }}:9443/oidc/endpoint/OP/user"
```

Comment:
This value should not contain the quotes.

Reply:
Since it is a string, I think it does not make any difference. Using quotes shd work.

Reply:
This is fine then